types/karma-sinon-chai/tsconfig.json

```diff
@@ -13,6 +13,11 @@
             "../"
         ],
         "types": [],
+        "paths": {
+            "@sinonjs/fake-timers": [
+                "sinonjs__fake-timers"
```

Comment:
are these paths additions because the package no longer provides its own types? were these definitions relying on shipped types before?

Reply:
Yes to both. This is basically just changing `package.json` `@sinonjs/fake-timers` to `@types/sinonjs__fake-timers`.